README.md

```diff
@@ -16,13 +16,48 @@ knife audit list
 knife audit show <id>
 knife notification list
 knife notification show <id>
+knife notification create <notification.json>
 knife rule list
 knife rule show <id>
+knife rule create <rule.json>
+```
+
+## Example data for creating rules and notifications:
+
+### Rules
+```
+{
+  "name": "New Rule Group 1",
+  "modified_by": "admin",
+  "with": {
+    "priority": 0
```

Comment:
it's weird having the priority here as well as in the rule. See https://github.com/chef/alaska_rules#rule-group-options
I can't remember why we do this, something to do with the front end.